web/styles/modal.css

```diff
@@ -67,6 +67,13 @@
         overflow: hidden;
         text-overflow: ellipsis;
     }
+
+    .user_profile_modal_pm {
+        margin-left: 9px;
+        display: flex;
+        align-items: center;
+        font-size: 17.3px;
```

Comment:
How did you pick 17.3px?

Reply:
> How did you pick 17.3px?

I chose a random value. I just discovered most of the icons have a font size of `18px` (for example, the edit icon in our own user profile modal), so I'll change the font size to `18px` for this particular CSS change.